### ISSUE TITLE
National Digital Catalogue Inventory

### ISSUE THREAD

**Issue by emilyodean** [NBMG-UNR/gbssrl_students#7]
Part of the 2019 NGGDPP funding available is for 77 student hours to be allocated towards an inventory of the NDC - https://www.sciencebase.gov/catalog/item/4f4e4760e4b07f02db47dfb4.

First, create an account. 
Then, go through each collection and make sure that the records online match the records in NBMG's spreadsheets.

Here is the text associated with this portion of the grant: 
"We propose an inventory and update of metadata resources available through the NDC in conjunction with our priority 2 project. The Great Basin Science Sample and Records Library (GBSSRL) houses a wealth of geoscience data and samples which have been collected by local, state, and federal agencies and the private sector. A number of physical holdings have been scanned to a digital format, associated with NGGDPP-compliant metadata, and uploaded to the NDC. The geoscience data manager for NBMG and GBSSRL will lead the effort required to inventory and quality assure all digital records that have been made available online. This effort will be assisted by a student worker in the geological sciences, and will involve updating metadata as necessary or removing obsolete entries."

**Comment by emilyodean** [comment]
A few things to note/investigate:

- It looks like there are two different datasets for mining district filse - "Collection of Mining District Files from Nevada" and "Nevada Bureau of Mines and Geology Nevada Mining District Files." These should be consolidated. Check which is the more recent and spot check to make sure it contains all the entries from the other one. Download and archive the other collection's csv, just in case. Also save the title and description, and archive all together in a zip file under G:/datasets/mining_district/archive with the title mdfiles_ndc_today'sdate.zip, and obviously replace "today'sdate" with today's actual date.
- Several collections are not associated with any records, and this is fine. We should at least have a collection title and description for every dataset that is in G:\datasets. Make sure to add the ones here that we don't have and write up a description. You may be able to find descriptions online on nbmg.unr.edu, by Googling the type of data, or by searching around the documentation on the G: drive.
- For other collections (engineering, Geothermal Well Files, Nevada geology, Oil and Gas, Sample Collection, Serial Publications), need to find the corresponding spreadsheet on the shared drives and compare the count of records. Remember to check for flags in the data that may indicate that it is proprietary. For mining district files, the metadata can be shared but not the documents themselves. For well logs, there's usually a period of time before data are released online.
- Update any collection descriptions with correct numbers from the spreadsheets on the G: drive.

**Comment by emilyodean** [comment]
@madsmiller since this is a large amount of work, make sure to be smart about how you do the comparisons between records. For example, you could download a record set from NDC, copy and paste the IDs from that file and the master spreadsheet into another file, and do a comparison in excel (https://chandoo.org/wp/compare-lists-excel-tip/). Definitely don't want you to have to look through and match each individual record - not only will that take a ton of time but also be more prone to error.

**Comment by madsmiller** [comment]
[National Digital Catalogue Inventory #7 Auditing.docx](https://github.com/NBMG-UNR/gbssrl_students/files/3958315/National.Digital.Catalogue.Inventory.7.Auditing.docx)

This is a list of discrepancies and things that need to be addressed for this project. It is being continually updated as I am looking through everything.

**Comment by madsmiller** [comment]
[National Digital Catalogue Inventory #7 Auditing.docx](https://github.com/NBMG-UNR/gbssrl_students/files/3975993/National.Digital.Catalogue.Inventory.7.Auditing.docx)

**Comment by madsmiller** [comment]
[National.Digital.Catalogue.Inventory.7.Auditing.docx](https://github.com/NBMG-UNR/gbssrl_students/files/4033017/National.Digital.Catalogue.Inventory.7.Auditing.docx)

**Comment by madsmiller** [comment]
[National.Digital.Catalogue.Inventory.7.Auditing (1).docx](https://github.com/NBMG-UNR/gbssrl_students/files/4037928/National.Digital.Catalogue.Inventory.7.Auditing.1.docx)

**Comment by emilyodean** [comment]
@madsmiller - Thanks for your great work on this! The information that you've captured is exactly what we need. I went through your document and commented on next steps - I highlighted 3 of the datasets that need some follow-up work to identify what is missing and update records. Feel free to work on them in whatever order you want. 

[National.Digital.Catalogue.Inventory.7.Auditing.1_emilycomments.docx](https://github.com/NBMG-UNR/gbssrl_students/files/4060458/National.Digital.Catalogue.Inventory.7.Auditing.1_emilycomments.docx)
